action.yml

```diff
@@ -2,27 +2,24 @@ name: 'Promote docker image'
 description: "Adds current branch tag and specified environment tag to a docker image. Determines source tag by examining PR that introduced the current commit."
 inputs:
   tag:
-    required: true
     description: New tag to apply to image
-    type: string
+    required: true
   add_ref_tag:
-    required: false
     description: Whether to apply a docker tag based on the current branch or github tag
-    type: boolean
```

Comment:
why is the type being removed?

Reply:
I'm not sure if it was like this from the beginning but there's no type for inputs in the schema [here](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs).

And the IDE validator also complains

![image](https://user-images.githubusercontent.com/1079045/205963629-7fed5821-4590-498a-bf56-0f0ce57ce864.png)